src/Hl7.Fhir.Specification/Specification/Snapshot/ElementDefnMerger.cs

```diff
@@ -300,12 +299,31 @@ internal FhirString mergeMax(FhirString snap, FhirString diff)
                     // do anything to not break it any further.
                     return snap;
                 }
-                else if (!diff.IsNullOrEmpty() && (snap.IsNullOrEmpty() || !diff.IsExactly(snap)))
+
+                if (!diff.IsNullOrEmpty() && (snap.IsNullOrEmpty() || !diff.IsExactly(snap)))
                 {
                     return deepCopyAndRaiseOnConstraint(diff);
                 }
-                else
-                    return snap;
+
+                return snap;
+            }
+
+            /// <summary>
+            /// Make sure max >= min
+            /// </summary>
```

Comment:
This method is not used (yet). Perhaps some extra documentation where it should be used.

Reply:
Done